### PR TITLE
Fix part of #1868: Fixed few TopicRevisionFragmentTest for Robolectric

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #1868 
Fixed few tests in `TopicRevisionFragmentTest`

Removed one test as it is e2e test which is not able to perform in Robolectric. We do have a way where we are keeping the required test in their particular file only, so the removed test is tested from it's own activity. 

Skipping 2 test cases:
`testTopicRevisionFragment_loadFragment_configurationChange_checkSpanCount_isThree`
`testTopicRevisionFragment_loadFragment_selectRevisionTopics_opensRevisionCardActivity`

## Roboelctric
Screenshot 2020-11-03 at 12 21 41

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/recyclerview/RecyclerViewMatcher.kt [hunk 1]
```diff
@@ -102,4 +106,30 @@ class RecyclerViewMatcher {
...
+      if (noViewFoundException != null) {
+        throw noViewFoundException
+      }
+      val recyclerView = view as RecyclerView
+      if (recyclerView.layoutManager is GridLayoutManager) {
+        val gridLayoutManager = recyclerView.layoutManager as GridLayoutManager
+        val spanCount = gridLayoutManager.spanCount
+        if (spanCount != expectedColumnCount) {
+          val errorMessage =
+            ("expected column count " + expectedColumnCount + " but was " + spanCount)
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/recyclerview/RecyclerViewMatcher.kt]: Nit:
`("expected column count $expectedColumnCount but was $spanCount")`

Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/recyclerview/RecyclerViewMatcher.kt]: @anandwana001 feel free to resolve on your own.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/sharedTest/java/org/oppia/android/app/recyclerview/RecyclerViewMatcher.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/recyclerview/RecyclerViewMatcher.kt]: Done

**Pull request merged**